src/borg/archiver/create_cmd.py

```diff
@@ -370,6 +371,10 @@ def _process_any(self, *, path, parent_fd, name, st, fso, cache, read_special, d
                     self.print_warning("Unknown file type: %s", path)
                     return
             except (BackupError, BackupOSError) as err:
+                if err is BackupOSError:
```

Comment:
I don't think this would work (guess you did not try?) as you are checking identity of a BackupOSError **instance** with the **class** - you need to use `isinstance()`.